### DISCUSSION TITLE
I can't access my benefits

### DISCUSSION THREAD

**Discussion by Halhadus** [community/community#73367] [GitHub Education]
### Hi everyone,

Hello. I submitted a Github student application on November 1st. My application was accepted and the attached image appeared. But I still cannot access my benefits. Is it normal? What should I do?
![Screenshot_20231104-074903_Fennec](https://github.com/community/community/assets/76776068/f66aaf8a-610d-405c-9a27-82d08f468675)

**Comment by GigaSmurf** [comment]
I think its normal for there to be a short delay. I believe I got a short delay as well before I could access stuff. Maybe it looks different on the computer, but probably not.

**Comment by Halhadus** [comment]
Problem solved.

**Reply by AntonC9018** [reply]
Could you share how?

**Comment by geroffss** [comment]
Same issue here. Got approved several days ago.
